StreetCare/StreetCare/Views/Screens/ICanHelpView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ICanHelpView.swift
+//  StreetCare
+//
+//  Created by Amey Kanunje on 9/27/24.
+//
+
+import SwiftUI
+
+struct ICanHelpView: View {
+    
+    @Binding var isPresented: Bool
+    @State private var isOutreachCreated = false
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+                    HStack {
+                        Spacer()
+                        Button(action: { isPresented = false }) {
+                            Image(systemName: "xmark")
+                                .foregroundColor(.gray)
+                        }
+                    }
+                    
+                    Text("Make sure you are not going alone")
+                        .font(.headline)
+                    
+                    Text("Group presence offers security and effectiveness in engaging with unfamiliar situations and individuals, benefiting both volunteers and the homeless.")
+                        .font(.subheadline)
+                        .foregroundColor(.secondary)
+                    
+                    Text("How outreach on Street Care works?")
+                        .font(.subheadline)
+                        .fontWeight(.semibold)
+                    
+                    Text("We post the outreach for you and other volunteers can sign up to go with you.")
+                        .font(.subheadline)
+                        .foregroundColor(.secondary)
+                    
+                    Button(action: {
```

Comment:
Same thing here there are three similar buttons with the same functionality could they be refactored into one?